app/Http/Middleware/Data.php

```diff
@@ -48,6 +48,12 @@ public function handle($request, Closure $next)
             return $page;
         }
 
+        // Overrides for meta information
+        $data['meta']['image'] = !empty($page['data']['meta_image']) ? $page['data']['meta_image'] : '';
```

Comment:
Thoughts on changing some of these long conditionals to use `Array::get()`?
https://laravel.com/docs/5.8/helpers#method-array-get

Something like:
`Array::get($page, 'data.meta_image', '');`